scripts/convert_hwp_previews.py

```
@@ -0,0 +1,113 @@
+    status_counts: Counter[str] = Counter()
+    candidates = 0
+    converted = 0
+    errors = 0
+
+    for path in sorted(raw_dir.glob("*.json")):
+        if source_prefix and not path.stem.startswith(source_prefix):
+            continue
+
+        try:
+            data = json.loads(path.read_text(encoding="utf-8"))
+        except Exception as exc:
```

Ruff (0.15.13)

[warning] Do not catch blind exception: `Exception`

(BLE001)